utils/lib/openqa_api.py

```diff
@@ -170,6 +170,10 @@ def get_job_combined_name(self):
         else:
             return self.job_name + '@' + self.machine
 
+    def get_job_test(self):
+        json_data = self.get_job_details()
+        return json_data['job']['settings']['TEST']
+
```

Comment:
This one is not needed (function not used anywhere, this file is used for reports reporting, not scheduling).